ddp-workspace/projects/ddp-dsm-ui/README.md

```diff
@@ -1,16 +0,0 @@
-# DSM (DDP Study Manager) Frontend
```

Comment:
Removed this since overall it was not accurate.  Moved the relevant bits into the README at the top level ddp-workspace.